src/ploneorg/addonlisting/browser/views.py

```diff
@@ -5,6 +5,7 @@
 from ploneorg.addonlisting.utils import update_addon_list
 from ploneorg.addonlisting.utils import update_addons
 from Products.Five.browser import BrowserView
+from plone.dexterity.browser.view import DefaultView
```

Comment:
I was planning to use DefaultView instead of BrowserView, when we require few extra properties that could be used in the template like view.w, view.widgets etc